SITnett/templates/meny.html

```diff
@@ -7,39 +7,45 @@
 	<meta charset="UTF-8"/>
 	<title>Studentersamfundets Interne Teater</title>
 	<link rel="stylesheet" href="{% static 'style.css' %}"/>
+	<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+	{% load static %}
+	<script src="{% static 'hamburger.js' %}"></script> 
```

Comment:
Siden `<script />`-elementet blokkerer innlastingen av siden kan vi vurdere å flytte det nederst i `<body />`. Usikker på om det har noe å si for funksjonaliteten, men dette skriptet er pittelite og igjen er nok dette noe som mest sannsynlig ikke har noenting å si.

Reply:
Ok, flytter det dit!